cfme/fixtures/multi_region.py

```diff
@@ -0,0 +1,75 @@
+import attr
+import pytest
+
+
+@attr.s
+class ApplianceCluster(object):
+    global_appliance = attr.ib(default=None)
+    remote_appliances = attr.ib(default=[])
+
+
+@pytest.fixture(scope='module')
+def multi_region_cluster(temp_appliances_unconfig_modscope_rhevm):
+    # todo: make it work for all appliance types
+    appliances = temp_appliances_unconfig_modscope_rhevm
+    cluster = ApplianceCluster()
+
+    cluster.global_appliance = appliances[0]
+    cluster.remote_appliances = appliances[1:]
+    yield cluster
+
+
+@pytest.fixture(scope='module')
+def setup_global_appliance(multi_region_cluster, app_creds_modscope):
+    global_app = multi_region_cluster.global_appliance
+
+    app_params = dict(region=99, dbhostname='localhost', username=app_creds_modscope['username'],
+                      password=app_creds_modscope['password'], dbname='vmdb_production',
+                      dbdisk=global_app.unpartitioned_disks[0])
+    global_app.appliance_console_cli.configure_appliance_internal(**app_params)
+    global_app.evmserverd.wait_for_running()
+    global_app.wait_for_web_ui()
+
+
+@pytest.fixture(scope='module')
+def setup_remote_appliances(multi_region_cluster, setup_global_appliance, app_creds_modscope):
+    remote_apps = multi_region_cluster.remote_appliances
+    gip = multi_region_cluster.global_appliance.hostname
+    for num, app in enumerate(remote_apps):
+        region_n = str(num + 1 * 10)
```

Comment:
Why `1*10`? Should this be `(num + 1) * 10`?

Reply:
multiplier always has higher priority than plus. 
so, first appliance will have region 10, second 20 and etc. why num+1 because first num is 0 :).